README.md

```diff
@@ -215,14 +217,76 @@ Endpoint  | Description
 `/health` | healthcheck - returns `OK`
 `/events` | event sink - returns `OK` if all keys are set in an event, error message otherwise
 
-### Known limitations
+## Advanced usage
```

Comment:
How about naming this section "Multiple ports support" since it describes how to do it.